examples/fluids/src/turb_spanstats.c

```diff
@@ -25,7 +25,7 @@ typedef struct {
   CeedElemRestriction elem_restr_parent_x, elem_restr_parent_stats, elem_restr_parent_qd, elem_restr_parent_colloc, elem_restr_child_colloc;
   CeedBasis           basis_x, basis_stats;
   CeedVector          x_coord, q_data;
-} * SpanStatsSetupData;
+} *SpanStatsSetupData;
```

Comment:
Yeah, I didn't like that either, but `clang-format` wanted the extra space.

Reply:
Yeah. I should update to a newer `clang-format` when I get a chance. My local is ahead of the CI version